bin/build-all.sh

```diff
@@ -11,32 +11,8 @@ if [ "$0" == "./bin/build-all.sh" -a -f index.ts ]; then
 
     echo ""
     echo '**********************************************'
-    echo "** Building taqueria-protocol"
-    npm run build --workspace=taqueria-protocol
-
-    echo ""
-    echo '**********************************************'
-    echo "** Building node-sdk"
-    npm run build --workspace=taqueria-sdk
-
-    echo ""
-    echo '**********************************************'
-    echo "** Building taqueria-state"
-    npm run build --workspace=taqueria-state
-
-    echo ""
-    echo '**********************************************'
-    echo "** Building taqueria-toolkit"
-    npm run build --workspace=taqueria-toolkit
-
-    for dir in `ls -1d *plugin*`; do
-        if [ -d $dir ]; then
-            echo ""
-            echo '**********************************************'
-            echo "** Building ${dir}"
-            ./bin/build-plugin.sh $dir
-        fi
-    done
+    echo "** Building packages"
+    npm run build --workspaces --if-present
```

Comment:
@GImbrailo a couple of thoughts here:

I like the simplicity of the single build command.

At this time, the taqueria CLI is in the root package.json. The root isn't considered a workspace, and thus we'll need to add a `npm run build` as well. 

I've always wanted to adjust the build script to: 
1) build packages in parallel (where possible);
2) show aggregate build success/failure messages post-building. 
3) halt building a package if any dependencies failed to build

I'm mentioning this because I'm not sure of how that will impact this script. Perhaps using a tool like the following would be sufficient, but I haven't had a chance to test it: `https://www.npmjs.com/package/workspaces-run/v/1.0.0`

Lastly, I have a concern about whether things get built in the correct order when not manually controlled. @alexzbusko perhaps you could try this? To do so:
1) Delete all build artifacts so that you have a clean state
2) Run `npm run build --workspaces --if-present`
3) Install the flextesa and taquito plugin
4) Try originating a contract

I'll try running through the same procedure a little later today.

--
Just while I'm thinking of it and to share those plans with you @Gino, there are some other adjustments I'd like to eventually make to the build process. I'll mention these things mostly for context, but if you think now might be a good time to roll some of these adjustments out, then perhaps that's something we could discuss:
- Taqueria CLI files should be moved to their own workspace. This will make editing in VSCode much easier, as we can switch between Deno and TypeScript language servers without having to open packages in separate windows
- Have build artifacts always go to a `dist` or `build` directory
- Have a `npm run clean` task to remove and clean up any built artifacts

Reply:
Thanks @mweichert. When packages will get built in the order in which they are defined in the `workspaces` directive of the `package.json` file. I think you raised this in that section of code but I will mention it here in one place. Previously we had `./taqueria*` and when we use globs `workspaces` does not do it in any specific order. Now we have specified so `workspaces` will build `./taqueria-protocol`, then `./taqueria-sdk` ans so on until it reaches line 67 where we still define `./taqueria*` asn it will build anything else that it has not built in previous steps.

You can find more information on this in the official [workspaces docs](https://docs.npmjs.com/cli/v7/using-npm/workspaces), specifically what is in the screenshot below
![image](https://user-images.githubusercontent.com/29157965/196230829-1e10026c-4e70-42c9-aebf-96d801fb4a5b.png)
 
I don't think the steps you gave @alexzbusko will work. The best is to run `npm run build all` as this will run in order:
```
npm install
npm run build --workspaces --if-present
npm run build
npm run build-docker --workspaces --if-present
```

Reply:
Also WRT the following:
>At this time, the taqueria CLI is in the root package.json. The root isn't considered a workspace, and thus we'll need to add a npm run build as well.

I did not remove the `npm run build` in the project root so it is not showing up in the changes but we are definitely still running a `build` in the root